workflow/log-monitoring/log-explorer.mdx

```diff
@@ -0,0 +1,79 @@
+
+# Log Properties
+
+The Log Properties section displays a componentized version of your raw JSON object: 
+
+![](https://lh7-us.googleusercontent.com/UEoX4SWceK7C5sfYo_BmTXKQjRV7bEOkEWOpo1NQPdFMKUkVOj0HaTJyg0SWkE_IYZWS7rID1aqBf7a4sOXcmIcMWuGKHR-3NPpGtIbk0tYgkWFOxIvTt0iVmr5QB9iHif9Z69If6ZPEYjJIj2cSCJY)
+
+# Raw
+
+The Raw section displays the JSON object for the specific log entry you are drilling into:
+
+![](https://lh7-us.googleusercontent.com/BL9KXXTTg-ckQ9aV1v_GxzInc6OTr8T62xS9Fp0Px4hCanVPth3fH7EGTx9pNZGxzWFLrWnPdktKNypYSRiINyctC4WEZRS0CmVm4Nyrj8pzAnNUxoPjDvobguPajmKXroCGjIRUNI_A4IW1Y-ViCec)
+
+# Metrics
+
+Within the Metrics section, there are three separate tabs relating to __Host__, __Process__, and __Container__ data.
+
+### Host
+
+The Host section displays all of your host data about the log you have drilled into. This tab is useful for identifying usage, network connection, and memory usage by state, and disk input/output on a per-host basis.
+
+![](https://lh7-us.googleusercontent.com/mH4R3eKAV72RzS4vAviQ70TfA1ThOEzJO4dSvXBf7oQ5eCqAcW5LZFFy_8ql-SuJNXNsBtFjV7OLx1xECE_G8NbJjPi6NkxNYcjIAAxz9Xw8GAsuDKpD8eRW1Uuu92QIxAco47RIWuIXZ-VziDNOEbk)
+
+### Process
+
+The Process section displays all the processes related to the log you have drilled into. This tab is useful for identifying CPU, physical, and virtual memory usage on a per-process basis.
+
+![](https://lh7-us.googleusercontent.com/KliZFaOP6bYef-DZ0C_hGxfb0--IHVEUlrgA70MZ6Cx_jEgxXM4REaQOfqlsRmF9bElrOCg5HFSQP9_z-kxrA7ku9dOool2fUoYs3bzjmapHSZntMYPgIXO5MXnt0d48NnnyGDtaT1snGoPsW8kOuCs)
+
+### Container
+
+The Container section displays all of your container data about the log you have drilled into. This tab is useful for identifying received and transmitted network packets as well as CPU usage on a per-container basis.
+
+![](https://lh7-us.googleusercontent.com/dZSRbZo7Xw0GqbFBc7_WkLqTgaGOsnVfLnbtsnMJUvjZZ5KIZsC_C7TU3zrsvL93wJWJwmNuz2RUXOs88zSxevlnY1BiXpElz13mo-DzxduryXzApT1sbdvwt8egSVIdDwwmN8JcU8TeETVFI3Q4p7E)
+
+# Source Logs
+
+The Source Logs section displays all of your unprocessed records generated by the underlying components of your infrastructure:
+
+![](https://lh7-us.googleusercontent.com/UJB8Qo60ar3HA7Qfg0afLkvOvvWgNL-eHCZbuyhkaTI1SE4F90klXkQZSxnFFV8KDU7hl9N5Q05hJRA_JMXaWNhyECUkdF3juEFBEdXftpfyJ9ZatrH3bb2ow8qixE3sov-5Pqd940yr6dLRvnSUkko)
+
+# Traces
+
+The Traces section displays all of your trace data related to the log you are drilling into. Logs contain an attribute called the trace ID, which correlates trace data to that specific log. Correlating logs with traces is useful for contextualizing larger system-wide issues within your infrastructure.
+
+Use the Flame Graph or Span View to understand the flow of trace requests through your system, the distribution of stack errors, and call hierarchies. The __View Service__ and __View Trace__ features (listed below) allow you to drill even further into your logs by contextualizing broader infrastructure issues.
+
```

Comment:
Recommend adding an info box that links to both APM Overview and RUM Overview pages to call out that this will not exist without first setting those up.